app/locations/none.js

```diff
@@ -1,16 +1,12 @@
-import Ember from 'ember';
-
-const {
-  computed,
-  computed: { bool, readOnly },
-  inject: { service },
-  get,
-  getOwner
-} = Ember;
+import { computed, get } from '@ember/object';
```

Comment:
I don't believe we can make these changes in `app/`. Sadly the codemod automatically does both `app` and `addon` directories, but within an addon we can only be certain that our own `addon/` (and `addon-test-support/`) are transpiled by a new enough ember-cli-babel version...

Reply:
We should probably move this code into `addon/` then re-export in `app`...